core/src/com/github/czyzby/tests/Main.java

```diff
@@ -234,6 +239,28 @@ public void process(final LmlParser parser, final LmlTag tag, final BlinkingLabe
         };
     }
 
+    //Provides tag for custom CodeTextArea
+    private static LmlTagProvider getCodeTextAreaProvider() {
+        return new LmlTagProvider() {
+            @Override
+            public LmlTag create(final LmlParser parser, final LmlTag parentTag, final String rawTagData) {
+                return getCodeTextAreaTag(parser, parentTag, rawTagData);
+            }
+        };
+    }
+
+    //Custom TextArea supporting embedding in scroll pane, used to display code of current LML example
+    private static LmlTag getCodeTextAreaTag(final LmlParser parser, final LmlTag parentTag, final String rawTagData) {
+        return new VisTextAreaLmlTag(parser, parentTag, rawTagData) {
+            @Override
+            protected VisTextField getNewInstanceOfTextField(TextLmlActorBuilder textBuilder) {
+                final CodeTextArea textArea = new CodeTextArea(textBuilder.getText(), textBuilder.getStyleName());
+                LmlUtilities.getLmlUserObject(textArea).setData(Boolean.TRUE); // Setting as multiline by default.
```

Comment:
`.setData(Boolean.TRUE)` is necessary to make text areas parse plain text between tags properly. Since there's no text between tags in main template, is is not necessary; `\n`  is already properly handled by the `TextArea` implementation. You can just return the actor: `return new CodeTextArea(textBuilder.getText(), textBuilder.getStyleName());`